Quaver.Shared/Screens/Gameplay/UI/PauseScreen.cs

```diff
@@ -135,9 +139,34 @@ public override void Update(GameTime gameTime)
             if (Screen.Failed)
                 Visible = false;
 
+            HandleInput();
+
             base.Update(gameTime);
         }
 
+        /// <summary>
+        ///     Handles all input for the screen.
+        /// </summary>
+        private void HandleInput()
```

Comment:
This shouldn't be handled here. It should be done in GameplayScreen.

https://github.com/Quaver/Quaver/blob/develop/Quaver.Shared/Screens/Gameplay/GameplayScreen.cs#L262